R/srp_model.R

```diff
@@ -53,11 +53,35 @@ create_srp_model <- function(
   attr(mdl, "states") <- c("stable", "response", "progression")
   attr(mdl, "visit_spacing") <- as.array(visit_spacing)
   attr(mdl, "stanmodel") <- stanmodels[["srp_model"]]
-  attr(mdl, "parameter_names") <- c("p", "shape", "scale", "median_time_to_next_event")
+  attr(mdl, "parameter_names") <- c("p", "shape", "scale",
+                                    "median_time_to_next_event")
   class(mdl) <- c("srp_model", "Model", class(mdl))
+  is_valid(mdl)
   return(mdl)
 }
 
+#' @import checkmate
+is_valid.srp_model <- function(mdl) { # nolint
+  # add double qoutes
```

Comment:
no dev comments please - either they explain something for future reference or delete them. Instead you could add a few comments below explaining the checks